src/document/operation/changeoperation.js

```diff
@@ -124,6 +97,35 @@ CKEDITOR.define( [ 'document/operation/operation', 'ckeditorerror' ], ( Operatio
 		getReversed() {
 			return new ChangeOperation( this.range, this.newAttr, this.oldAttr, this.baseVersion + 1 );
 		}
+
+		clone() {
+			return new ChangeOperation( this.range.clone(), this.oldAttr, this.newAttr, this.baseVersion );
+		}
+
+		/**
+		 * Checks whether this operation has conflicting attributes with given {@link document.operation.ChangeOperation}.
+		 * This happens when both operations changes an attribute with the same key and they either set different
+		 * values for this attribute or one of them removes it while the other one sets it.
+		 *
+		 * @param {document.operation.ChangeOperation} otherOperation Operation to check against.
+		 * @returns {boolean} True if operations have conflicting attributes.
+		 */
+		conflictsAttributesWith( otherOperation ) {
```

Comment:
`conflictsAttributesWith` should got to `transformoperation`. On the other hand we could have helper to compare two attributes and better handle the "null" operation. I met similar problem creating `ChangeDelta`: https://github.com/ckeditor/ckeditor5-core/blob/master/src/document/delta/changedelta.js#L144-L145.I think it will be changed when we will close https://github.com/ckeditor/ckeditor5-core/issues/63, so we could leave comment there and fix it later.